### ISSUE TITLE
Add support for equals matrix

### ISSUE THREAD

**Issue by rusu24edward** [rusu24edward/MatMan#16]
Add support for equals operator and function to set the matrix part in the LHS to the matrix part from the RHS Do this for arbitrary submatrix selection.

**Comment by rusu24edward** [comment]
Blocked by MatMan-23

**Comment by rusu24edward** [comment]
Won't do. This feature is better covered by MatMan-40
